README.md

```diff
@@ -107,12 +107,206 @@ See
 
 # Methods
 
+# Budgets and Billing Tags
+
+See
+[`example_budget_manager.py`](https://github.com/dominodatalab/python-domino/blob/release-1.3.1/examples/example_budget_manager.py)
+for example code.
+
+### budget_defaults_list()
+
+Get a list of the available default budgets with the assigned (if any) limits
+Requires Admin permission
+
+### budget_defaults_update(budget_label, budget_limit)
+
+Update default budgets by BudgetLabel
+Requires Admin roles
+
+-   *budget_label:* (required) label of budget to be updated ex: `BillingTag`, `Organization`
+
+-   *budget_limit:* (required) new budget quota to assign to default label
+
+### budget_overrides_list()
+
+Get a list of the available budgets overrides with the assigned limits.
+Requires Admin permission
+
+### budget_override_create(budget_label, budget_id, budget_limit)
+
+Create Budget overrides based on BudgetLabels, ie BillingTags, Organization, or Projects
+the object id is used as budget ids
+Requires Admin roles
+
+-   *budget_label:* label of budget to be updated
+
+-   *budget_id:* id of project or organization to be used as new budget override id.
+
+-   *budget_limit:* budget quota to assign to override
+
+### budget_override_update(budget_label, budget_id, budget_limit)
+
+Update Budget overrides based on BudgetLabel and budget id
+Requires Admin roles
+
+-   *budget_label:* label of budget to be updated
+
+-   *budget_id:* id of budget override to be updated.
+
+-   *budget_limit:* new budget quota to assign to override
+
+### budget_override_delete(budget_id)
+
+Delete an existing budget override
+Requires Admin roles
+
+-   *budget_id:* id of budget override to be deleted.
+
+### budget_alerts_settings()
+
+Get the current budget alerts settings
+Requires Admin permission
+
+### budget_alerts_settings_update(alerts_enabled, notify_org_owner)
+
+Update the current budget alerts settings to enable/disable budget notifications
+and whether to notify org owners on projects notifications
+Requires Admin permission
+
+-   *alerts_enabled:* whether to enable or disable notifications.
+
+-   *notify_org_owner:* whether to notify organizations owners on projects reaching threshold.
+
+### budget_alerts_targets_update(targets)
+
+Update the current budget alerts settings with additional email targets per budget label
+Requires Admin permission
+
+-   *targets:* dictionary of budget labels and list of email addresses
+
+### active_billing_tags_list()
```

Comment:
How do you feel about `billing_tags_list_active`?  It would be kinda nice if all the billing tag methods started with `billing_tag`, although I don't think this is a big deal.

Reply:
that works